packages/tempus/src/Tempus_StepperLeapfrog_decl.hpp

```diff
@@ -12,7 +12,11 @@
 #include "Tempus_config.hpp"
 #include "Tempus_StepperExplicit.hpp"
 #include "Tempus_StepperObserverComposite.hpp"
-#include "Tempus_StepperLeapfrogObserver.hpp"
+#ifndef TEMPUS_HIDE_DEPRECATED_CODE
+  #include "Tempus_StepperLeapfrogObserver.hpp"
+#endif
+#include "Tempus_StepperLeapfrogAppAction.hpp"
+#include "Tempus_StepperLeapfrogAppActionComposite.hpp"
```

Comment:
Let's go ahead and remove the *Composite* code.  The Stepper should only need to know about the AppAction, and not whether if it is an AppAction composite.